components/LineChart.stories.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import LineChart from './LineChart';
+
+export default {
+  title: 'LineChart',
+  component: 'LineChart',
+};
+
+const data = [
```

Comment:
I am curious how `LineChart` would behave if 
1. there are holes in time (i.e. skipped date)
2. contains data with less than 30 days

Is it going to be handled in `LineChart` or is it going to be handled in some wrapper component?

Reply:
I believe we discussed this briefly at moed23ct, the ultimate solution is to populate data for all missing dates for the last 30 days.
The plan is to move the dataset handling part inside `getChartData` to the parent component and we can handle missing data there as well.
I think this component should just plot the charts with given data and layout

Reply:
Got it, agree that we can just plot the data given to `<LineChart>` to keep it simple. Thanks for the clarification!